webapp/sources/rudder/rudder-rest/src/main/scala/com/normation/rudder/web/lift/JsCommands.scala

```diff
@@ -0,0 +1,22 @@
+package com.normation.rudder.web.lift
+
+import net.liftweb.http.js.JsCmd
+import scala.xml.Node
+import scala.xml.Unparsed
+
+object JsCommands {
+
+  // copied and adapted from Lift JsCommands code
```

Comment:
Shouldn't imports be a list if path that we check from path traversal and we rebuild the correct relative path from a directory? Having .. In imports is unsetling